types/ts3-nodejs-library/TeamSpeak3.d.ts

```diff
@@ -0,0 +1,990 @@
+import { EventEmitter } from 'events';
+import TeamSpeakChannel = require('./property/Channel');
+import TeamSpeakClient = require('./property/Client');
+import TeamSpeakServer = require('./property/Server');
+import TeamSpeakServerGroup = require('./property/ServerGroup');
+import TeamSpeakChannelGroup = require('./property/ChannelGroup');
+
+// For the buffer type.
+import * as stream from 'steam';
```

Comment:
This might the what's causing your dependency. I think you meant `stream`

Reply:
Oh yeah that’s a typo. I will fix it in a new pull request after I added more typings.